app/models/miq_group.rb

```diff
@@ -60,8 +60,8 @@ def settings=(new_settings)
     super(indifferent_settings)
   end
 
-  def self.with_allowed_roles_for(user_or_group)
-    includes(:miq_user_role).where.not({:miq_user_roles => {:name => user_or_group.disallowed_roles}})
+  def self.with_roles_excluding(disallowed_roles)
```

Comment:
Are all of the callers for this updated?  I don't see any changed here.

Reply:
Thankfully these ones got hidden due to recent commits.  Just noticed where these were being used, so this question and others like it can be ignored.